lib/pdn/pdn.c

```diff
@@ -117,6 +118,37 @@ static void on_cnec_esm(const char *notif)
 	}
 }
 
+static void on_cnec_emm(const char *notif)
+{
+	char *p;
+	uint32_t cid = 0;
+	uint32_t emm_err = UINT32_MAX;
+	struct pdn *pdn;
+	int ret;
+
+	/* +CNEC_EMM: <cause>,<cid> */
+	ret = sscanf(notif, "+CNEC_EMM: %d,%d", emm_err, cid);
```

Comment:
This is incorrect and most likely causes a crash. This also causes two compilation warnings, so it was easy to spot.
```suggestion
	ret = sscanf(notif, "+CNEC_EMM: %d,%d", &emm_err, &cid);
```